CHANGELOG.md

```diff
@@ -20,71 +20,77 @@
 
 Individual unreleased changelog entries can be located at [changelog/unreleased](changelog/unreleased). They will be assembled into [CHANGELOG.md](CHANGELOG.md) once released.
 
-## 3.8.0
```

Comment:
We should keep the 3.8.0 index. Also, we should increase the indent as the original format.